bazel/repositories_extra.bzl

```diff
@@ -17,6 +17,7 @@ def envoy_dependencies_extra(python_version = PYTHON_VERSION):
     python_register_toolchains(
         name = "python%s" % ("_".join(python_version.split(".")[:-1])),
         python_version = python_version,
+        ignore_root_user_error = True,  # TODO: Change to non-root user in docker image
```

Comment:
small nit: we already have this fixed - its just downstreams that need to do this

Reply:
I actually just removed it instead, lmk if you think we should keep it though. But I imagine we don't care that much to push on this